main.tf

```diff
@@ -221,10 +222,10 @@ resource "aci_rest_managed" "l3extRsPathL3OutAtt" {
   class_name = "l3extRsPathL3OutAtt"
   content = {
     addr       = each.value.ip
-    autostate  = "disabled"
     descr      = each.value.description
     encapScope = "local"
     ifInstT    = each.value.vlan != null ? (each.value.svi == "yes" ? "ext-svi" : "sub-interface") : "l3-port"
+    autostate  = each.value.autostate ? "enabled" : "disabled"
```

Comment:
As the conversion is already done under `local.interfaces` there is no need to do it again.